src/infra/virtualHostCmdRepo.go

```diff
@@ -75,6 +75,83 @@ func (repo VirtualHostCmdRepo) addAlias(addDto dto.AddVirtualHost) error {
 	return repo.reloadWebServer()
 }
 
+func (repo VirtualHostCmdRepo) addPhpVirtualHost(hostname valueObject.Fqdn) error {
+	phpVhostConfFilePath, err := RuntimeQueryRepo{}.GetVirtualHostPhpConfFilePath(hostname)
+	if err != nil && err.Error() != "VirtualHostNotFound" {
+		return err
+	}
+
+	vhostsWithMappings, err := VirtualHostQueryRepo{}.GetWithMappings()
+	if err != nil {
+		return err
+	}
+
+	var targetVhostWithMappings dto.VirtualHostWithMappings
+	for _, vhostWithMapping := range vhostsWithMappings {
+		if vhostWithMapping.Hostname.String() != hostname.String() {
+			continue
+		}
+
+		targetVhostWithMappings = vhostWithMapping
+		break
+	}
+
+	shouldCreatePhpVhostConf := true
+	for _, targetVhostMapping := range targetVhostWithMappings.Mappings {
+		isServiceMapping := targetVhostMapping.TargetType.String() == "service"
+		isPhpService := targetVhostMapping.TargetServiceName.String() == "php"
+		if isServiceMapping && isPhpService {
+			shouldCreatePhpVhostConf = false
+		}
+	}
+
+	if !shouldCreatePhpVhostConf {
+		return nil
+	}
```

Comment:
What is this dude?

Reply:
Retirei isso e to usando apenas o `GetVirtualHostPhpConfFilePath` agora.